### PR TITLE
Changes following first CRAN submission

### PR DESCRIPTION
**Pull request by jameslamb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,10 +23,10 @@ The core functionality of this package is the `es_search` function. This returns
 
 ### R <a name="rinstallation"></a>
 
-The package is not yet on CRAN, but we'll be submitting soon. For now, you can install using `devtools`.
+This package can be installed directly from CRAN.
```
Comment (terrytangyuan) [inline README.md]: We should also keep the instruction for installing dev version for adventurous users

Comment (jameslamb) [inline README.md]: Good idea! Will fix

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # uptasticsearch
 
+[![CRAN\_Status\_Badge](http://www.r-pkg.org/badges/version-last-release/uptasticsearch)] [![CRAN\_Download\_Badge](http://cranlogs.r-pkg.org/badges/grand-total/uptasticsearch)]
```
Comment (austin3dickey) [inline README.md]: Can you add the badges in the `r-pkg/README` as well and regen `/docs/` again?

Comment (austin3dickey) [inline README.md]: Hopefully pkgdown will add the badges to its sidebar; that's the intended feature

**Pull request merged**